db/migrate/20200726044317_create_watched_buttons.rb

```diff
@@ -0,0 +1,11 @@
+class CreateWatchedButtons < ActiveRecord::Migration[6.0]
+  def change
+    create_table :watched_buttons do |t|
+      t.string :user_email
+      t.string :movie_title
```

Comment:
`user_email` になっていますが， どのユーザーの視聴済み情報かを判定したいならば，必要となるのは `user_id` です。

どの動画が視聴済みなのかを判定するなら必要となるのは `movie_id` です。

```suggestion
      t.references :user, null: false, foreign_key: true
      t.references :movie, null: false, foreign_key: true
```

さらに，モデルだけでなく，テーブルにも `user_id` と `movie_id` の組みが複数作成されないようにしておくべきです。